app/controllers/comments_controller.rb

```diff
@@ -1,24 +1,23 @@
-class CommentsController < ApplicationController
+# frozen_string_literal: true
```

Comment:
remove these comments if you don't need those

Reply:
removed bhai